docker-deploy.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -ex
```

Comment:
Sounds good, can you push that change? I'd really like to "contain" all the docker-related stuff in just one place. Especially since docker is rapidly changing/unstable and I don't want to go looking through several different files to maintain it. :)